view/pe/peview.cpp

```diff
@@ -597,15 +597,9 @@ bool PEView::Init()
 		m_simplifyTemplates = viewSettings->Get<bool>("analysis.types.templateSimplifier", this);
 
 		settings = GetLoadSettings(GetTypeName());
-		if (settings && settings->Contains("loader.imageBase") && settings->Contains("loader.architecture")) // handle overrides
-		{
+		if (settings && settings->Contains("loader.imageBase")) // handle overrides
 			m_imageBase = settings->Get<uint64_t>("loader.imageBase", this);
 
-			Ref<Architecture> arch = Architecture::GetByName(settings->Get<string>("loader.architecture", this));
-			if (!m_arch || (arch && (arch->GetName() != m_arch->GetName())))
-				m_arch = arch;
```

Comment:
m_arch is not being set based on the overridden platform. It is set based on the original platform, which may have changed due to an override.